.storybook-s2/docs/Migrating.jsx

```diff
@@ -0,0 +1,770 @@
+import { style } from '../../packages/@react-spectrum/s2/style/spectrum-theme' with {type: 'macro'};
+import {Link} from '@react-spectrum/s2';
+import {P, Code, Pre, H3, H2} from './typography';
+
+export function Migrating() {
+  return (
+    <div className={style({maxWidth: 'lg', marginX: 'auto', fontFamily: 'sans'})}>
+      <div className={style({marginX: 48})}>
+        <h1 className={style({fontSize: 'heading-2xl', color: 'heading', marginBottom: 48})}>
+          Migrating to Spectrum 2
+        </h1>
+        <P>An automated upgrade assistant is available by running the following command in the project you want to upgrade:</P>
+        <Pre>npx @react-spectrum/codemods s1-to-s2</Pre>
```

Comment:
Does the `use-monopackage' code mod need to be documented as well? Or do we expect it to be pretty self explainatory/not used as much/can rely on the README for that one?

Reply:
Not sure of a good place for it. Maybe here? https://react-spectrum.adobe.com/react-spectrum/versioning.html

Reply:
Yeah that seems like a good place. If you'd like we can defer if we think people will mainly use the `s1-to-s2` code mod